app/assets/javascripts/channels/circle_check_scores.coffee

```diff
@@ -4,5 +4,5 @@ App.circle_check_scores =
       participant = data.participant_id
       score = data.score
       cell = $("tr[data-participant-id=#{participant}] td.circle-check-score")
-      cell.text(score).attr('data-text', score).attr('data-score', score)
+      cell.text(score).attr('data-text', score).attr('data-score', score).addClass('last-updated')
```

Comment:
To ensure it triggers on a second update, you need to remove and re-add the class so the animation will trigger again. Perhaps a method for this will help? If it's an infrequent use case then maybe too much work.

Reply:
I feel like there will be opinions about what I did... but it works!